recipe/meta.yaml

```diff
@@ -18,7 +18,7 @@ requirements:
 
   build:
     - python
-    - setuptools
+    - setuptools =20.7.0
```

Comment:
So, no `=` needed when pinning. Also, turns out the fix is to add this `vs2015_runtime 14.00.23026.0 0 # [win and py35]`, which needs to go in run requirements too. A new build of `vs2015_runtime` was released in the last few hours, which caused the Python 3.5 failure.